### ISSUE TITLE
Footnote Quickjump not appearing

### ISSUE THREAD

**Issue by Blackhawke** [typora/typora-issues#4504]
**Describe the bug**

The quickjump link doesn't seem to reliably appear. Usually, on opening the file they don't. If a file is opened from another file they do, until you reload the file. Then they vanish.

**Expected behavior**
I'm assuming the quickjump links are supposed to be present at all times.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04/Gnome

**Typora Version**
e.g: 0.10.11

**Additional context**
Nothing more to add. Pretty simple (for a change)

**Comment by abnerlee** [comment]
> If a file is opened from another file they do, until you reload the file. Then they vanish.

Could you attach a screencast? I'm not able to reproduce this.

**Comment by Blackhawke** [comment]
> > If a file is opened from another file they do, until you reload the file. Then they vanish.
> 
> Could you attach a screencast? I'm not able to reproduce this.

Certainly. Here's the file on opening. You can see the footnote return links.
![Screenshot from 2021-07-10 11-20-07](https://user-images.githubusercontent.com/1757829/125173298-72e48680-e173-11eb-9dae-82a887707a1e.png)
Now, here's same footnotes shown _after_ reloading the file. Note that the return links are gone.
![Screenshot from 2021-07-10 11-26-25](https://user-images.githubusercontent.com/1757829/125173316-927baf00-e173-11eb-8a89-9e13a92b7fdd.png)

**Comment by Blackhawke** [comment]
Certainly, though I will say, in further testing, I've also found it hard to duplicate. Which makes me wonder what is in the files that's causing the problem. We sometimes work on our Android Tablets and sometimes on our desktops, but I've looked at several files now, all in our git repo, and some cause the behavior. Most don't. I also created a new file in Typora, then edited it with Gedit and reloaded the file using `files -> reload with encoding -> auto` (you don't have a reload, nor does the editor auto-detect changes, so...) and it doesn't do what I'm about to show you in the images.

First image, freshly loaded, in this case from a link in another file in Typora. The return footnote links are present.

![Screenshot from 2021-07-10 12-00-37](https://user-images.githubusercontent.com/1757829/125174103-cfe23b80-e177-11eb-8937-6b5562bf647c.png)

Here's the same file, reloaded using `files -> reload with encoding -> auto`.
![Screenshot from 2021-07-10 12-03-38](https://user-images.githubusercontent.com/1757829/125174121-ea1c1980-e177-11eb-9580-31f7c819f90d.png)

Note that the return links are gone.

I don't know why it's doing this, in some files and not others. It's very strange.
